### ISSUE TITLE
Peer dependency version range claims impossible supported range

### ISSUE THREAD

**Issue by Gerrit0** [typhonjs-typedoc/ts-lib-docs#1]
package.json claims:

> ```
>   "peerDependencies" : {
>    "typedoc": ">=0.25.0 || 0.26.x || 0.27.x || 0.28.x || 0.29.x || 0.30.x || 0.31.x"
>  },
> ```

This can't possibly be correct as 0.28 doesn't exist yet, and is guaranteed to break this plugin as it uses `app.renderer.theme.getUrls(project);` which is going away in that version

**Comment by typhonrt** [comment]
I just put the extraneous fields in there because the automatic TypeDoc plugin listing does not handle `>=0.25.0` to list it in 0.26, 0.27 plugin listings; it stays less visible in `0.25` plugins. 

This package as a plugin just does the unknown symbol link resolution to the hosted TS lib docs, so there is no worry about `app.renderer.theme.getUrls` being used by plugin consumers. I'll have to worry about `app.renderer.theme.getUrls` next time I update ts-lib-docs doc generation at the end of next year or thereabout; whenever the built-in TS libs hit ES2025, etc.

But yes... doing `app.renderer.theme.getUrls` was a missing step to make sure the data generated for URL linking is correct. The way it was before didn't pick up the kind sort order used for the docs generation in respect to the URL to reflection mapping. I think I got that working right. It's a lot of data to try and verify.

I also need to compress the URL lookup JSON as it's ~10MB per year of docs generated next time around.

**Comment by Gerrit0** [comment]
Yeah... typedoc's site doesn't list plugins which claim unrealistic supported versions. if it is just the plugins under "external" which are actually published, then it's *probably* safe... I don't currently have any plans to change that API (though I'm less confident about where it is located, not convinced converter was the right place for it)

**Comment by typhonrt** [comment]
It does look like things posted correctly in the TypeDoc plugin listing. I will make another prompt release of `ts-lib-docs` if any problems arise in coming TypeDoc releases.
